go-controller/pkg/networkmanager/nad_controller.go

```diff
@@ -160,8 +161,12 @@ func (c *nadController) syncAll() (err error) {
 			continue
 		}
 		err = c.syncNAD(key, nad)
-		if err != nil {
-			return fmt.Errorf("%s: failed to sync %s: %v", c.name, key, err)
+		if err != nil && !errors.Is(err, config.ErrorAttachDefNotOvnManaged) {
+			if c.recorder != nil {
+				c.recorder.Eventf(&corev1.ObjectReference{Kind: nad.Kind, Namespace: nad.Namespace, Name: nad.Name}, corev1.EventTypeWarning,
+					"FailedSync", "Failed to sync network %s config: %v", key, err)
+			}
+			klog.Warningf("%s: unable to sync %s: %v", c.name, key, err)
```

Comment:
nit:
```suggestion
			klog.Warningf(" nadController %s unable to sync NAD %s: %v", c.name, key, err)
```

Reply:
This is what is written to `c.name`:
`name:              fmt.Sprintf("[%s NAD controller]", name),`